operator/pkg/k8s/constants.go

```diff
@@ -45,7 +45,7 @@ const (
 	UpToDate                             = "UpToDate"
 	UpToDateMessage                      = "ConfigConnector is up to date"
 	UpdateFailed                         = "UpdateFailed"
-	ReconcileErrMsgTmpl                  = "error during reconciliation: %v"
+	ReconcileErrMsgTmpl                  = "error during reconciliation: %w"
```

Comment:
My _personal_ view is that making error messages into constants is just going too far on the whole "no duplicated values" concept :-)

Reply:
I can get behind removing the template 👍🏼